presto-main/src/main/java/io/prestosql/sql/planner/planprinter/IoPlanPrinter.java

```diff
@@ -94,6 +100,22 @@ public IoPlan(
         {
             this.inputTableColumnInfos = ImmutableSet.copyOf(requireNonNull(inputTableColumnInfos, "inputTableColumnInfos is null"));
             this.outputTable = requireNonNull(outputTable, "outputTable is null");
+            long totalEstBytesScanned = 0;
+            boolean hasTotalEstBytesScanned = false;
+            for (TableColumnInfo info : inputTableColumnInfos) {
+                Long estBytesScanned = info.getEstBytesScanned();
+                if (estBytesScanned != null) {
```

Comment:
what if some `TableColumnInfo` have this field and some don't? Does it make sense to calculate `totalEstBytesScanned `, is it accurate enough?

Reply:
For our use case, the purpose of the estimates output are to get a rough idea of the relative expense of the query The estimate are purely based on statistics generated by the connectors, hence their accuracy cannot be guaranteed. It's really more of a best effort.